Source/WebCore/dom/Element.cpp

```diff
@@ -3222,13 +3222,13 @@ static RefPtr<Element> autoFocusDelegate(ShadowRoot& target)
 }
 
 // https://html.spec.whatwg.org/multipage/interaction.html#focus-delegate
-static RefPtr<Element> focusDelegateFromShadowHost(ShadowRoot& target)
+static RefPtr<Element> focusDelegateFromContainerNode(ContainerNode& target)
```

Comment:
Why not calling this function findFocusDelegateInternal or findFocusDelegateRecursively.